src/NeoSharp.Core/SmartContract/Invocation/InvocationResult.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using NeoSharp.Core.Types;
+using NeoSharp.VM;
+
+namespace NeoSharp.Core.SmartContract
+{
+    public class InvocationResult
+    {
+        public readonly byte[] ExecutedScript;
+        public readonly EVMState ResultingState;
+        public readonly Fixed8 GasCost;
+        public readonly IStackItemsStack ExecutionResult;
```

Comment:
Maybe is easier to get something like `object[]` as result, because on NeoVM you need to release / dispose this items for free the memory, so maybe is better convert before IntegerStackItem -> Int and return the corresponding values